### ISSUE TITLE
Keyboard focus in Bottom and docked Sheets is not implemented properly

### ISSUE THREAD

**Issue by JayPanoz** [edrlab/thorium-web#66]
Unfortunately it seems that using React Aria’s `FocusScope` in  is not enough for keyboard navigation in the bottom Sheet (using React Modal Sheet) and dockedSheets. 

In the case of the bottom sheet, it won’t automatically scroll the focused element into view like it does for vanilla components such as `Popover` for instance. Not sure it’s because of React Modal Sheet’s `Scroll` element, or if it’s to be expected, I’ll have to test. 

In the case of docked sheets, it scrolls the entire document. Although I guess this is more or less a CSS issue first and foremost.

**Comment by JayPanoz** [comment]
So Docked sheets was a CSS issue. 

~~Bottom Sheet is quite another issue though.~~

Nevermind, bottom sheet was a CSS issue as well – it was just missing a `position: relative`

**Comment by JayPanoz** [comment]
Note to self: RAC `FocusScope` needs `position: relative | absolute` to work properly and scroll focus elements into view.
